modules/sdk-coin-sol/src/lib/instructionParamsFactory.ts

```diff
@@ -59,6 +59,8 @@ export function instructionParamsFactory(
       return parseAtaInitInstructions(instructions);
     case TransactionType.StakingAuthorize:
       return parseStakingAuthorizeInstructions(instructions);
+    case TransactionType.StakingAuthorizeRaw:
+      return parseStakingAuthorizeInstructions(instructions);
```

Comment:
For this parse, shouldn't we also throw if the # of instructions and that each instruction is for an expected program?

Reply:
good catch, this should be removed as I moved the new builder away from TransactionBuilder but BaseTransactionBuilder